src/work-allocation/components/task-assignment/task-assignment.component.ts

```diff
@@ -159,4 +205,11 @@ export class TaskAssignmentComponent implements OnInit {
     return toVet;
   }
 
+  // sets the selected location on the location dropdown
+  public setSelectedLocation(): void {
```

Comment:
Same comments as the `locationMatches` method - it _looks_ like you're actually choosing the location of the assigned caseworker, not just generically setting a selected location. My initial reading of this method was that it was getting called from the HTML or something but then I realised it also didn't have a parameter.

On that subject - `setX` methods should take a parameter with the value you're trying to set a property to. I think you're looking for `setupX` in this instance.

Reply:
Yep, will change to setupSelectedLocation()